demos/compile_demo/compile_templates.js

```diff
@@ -0,0 +1,6 @@
+var exec = require('child_process').exec;
+var cmd = 'twigjs --output compiled/ --pattern *.twig raw/';
+
+exec(cmd, function(error, stdout, stderr) {
```

Comment:
Since this is JavaScript, you could probably accomplish this without needing to invoke a child process :wink: